src/relay/transforms/fuse_ops.cc

```diff
@@ -1017,18 +1050,30 @@ class FuseMutator : private MixedModeMutator {
   }
 };
 
-Expr FuseOps(const Expr& expr, int fuse_opt_level, size_t max_fuse_depth, const IRModule& module) {
-  return FuseMutator().Transform(expr, fuse_opt_level, max_fuse_depth);
+Expr FuseOps(const Expr& expr, int fuse_opt_level, size_t max_fuse_depth, bool link_params,
+             const IRModule& module) {
+  return FuseMutator()
```

Comment:
Could we use a constructor to set these values ? -- Insread of calling a chain of Setters.

Reply:
Done